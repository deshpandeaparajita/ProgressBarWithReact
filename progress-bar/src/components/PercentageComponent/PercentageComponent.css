
.completed {
    height: 100%;
    border-radius: inherit;
    transition: width .3s ease-in;
}

