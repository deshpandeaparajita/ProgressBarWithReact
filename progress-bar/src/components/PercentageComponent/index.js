export {default as PercentageComponent} from './PercentageComponent';