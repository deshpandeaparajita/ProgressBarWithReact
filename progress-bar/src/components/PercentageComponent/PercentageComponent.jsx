import React from 'react';
import './PercentageComponent.css';


export const PercentageComponent = (props) => {
    
    return (
        props.percentRange === 100 ? (
            <div className="completed" style={{ width: `${props.percentRange}%`, background: 'red' }} />
        ) : (
            <div className="completed" style={{ width: `${props.percentRange}%`, background: 'limegreen' }} />
        )


    );
};

export default PercentageComponent;