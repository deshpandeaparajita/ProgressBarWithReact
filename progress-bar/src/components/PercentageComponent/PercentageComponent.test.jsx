import React from 'react';
import { shallow } from 'enzyme';
import PercentageComponent from './PercentageComponent';



describe('PercentageComponent', () => {
    it('should match snapshot', () => {
        const component = shallow(<PercentageComponent />);
        expect(component).toMatchSnapshot();
    });
});