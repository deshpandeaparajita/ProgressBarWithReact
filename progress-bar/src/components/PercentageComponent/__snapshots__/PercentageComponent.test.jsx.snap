// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PercentageComponent should match snapshot 1`] = `ShallowWrapper {}`;
